pkg/controller/volume/attachdetach/cache/actual_state_of_world.go

```diff
@@ -616,8 +616,7 @@ func (asw *actualStateOfWorld) GetAttachedVolumesForNode(
 	asw.RLock()
 	defer asw.RUnlock()
 
-	attachedVolumes := make(
-		[]AttachedVolume, 0 /* len */, len(asw.attachedVolumes) /* cap */)
+	attachedVolumes := make([]AttachedVolume, 0)
```

Comment:
Em, as if it is reasonable. But it's best for you to submit an issue to confirm the necessity of this change, I think.